packages/nexrender-worker/src/index.js

```diff
@@ -84,9 +85,19 @@ const start = async (host, secret, settings) => {
         } catch (err) {
             job.retries = job.retries || 0
 
-            if (job.retries < NEXRENDER_MAX_RETRIES) {
+            if (job.retries <= NEXRENDER_MAX_RETRIES) {
                 job.retries += 1;
                 job.state = 'queued';
+
+                // Remove re-queued jobs as they may be picked up
+                // by the same worker
+                // sometimes this attribute (workpath) is undefined
+                if (!job.workpath) {
+                    job.workpath = settings.workpath.concat('/', job.uid, '/')
+                }
+                await new Promise((resolve) => {
```

Comment:
Does we need to silently catch errors here?

Reply:
We could log errors actually. I'll add.

Reply:
Done